### DISCUSSION TITLE
How can I increase the size of the Erasure Code Parity to the size of nodes - 1?

### DISCUSSION THREAD

**Discussion by xixici** [minio/minio#17910] [Q&A] [answered]
Why do I want such an effect? 
It's because I want to achieve full replication for zero tolerance towards data loss. 
My idea might be a bit elementary, so I hope you won't mock me.

I would greatly appreciate any alternative methods that can achieve the same effect. Thank you very much.

**Comment by harshavardhana** [comment, marked as answer]
You are asking for EC:0 - this is supported already.

**Reply by xixici** [reply]
Thanks. 
But, according to my understanding, setting EC:0 implies disabling erasure coding, which means no redundancy is provided and reliability is lost. 
What I'm aiming for is maximum redundancy, so that even if multiple nodes fail, as long as one is operational, my data remains intact.

**Reply by harshavardhana** [reply]
that is governed via how many nodes you have and how many parity you have.

**Reply by xixici** [reply]
I apologize for the misunderstanding. I have 6 nodes and I want to ensure that even if any 5 of them fail, the data remains intact.

**Reply by harshavardhana** [reply]
that is not possible @xixici we do not support traditional disk replication if that's what you are looking for.
